Makefile

```diff
@@ -40,7 +40,7 @@ some_recipe:
 test: test-unit test-main
 
 .PHONY: pre-pr
-pre-pr: lint test
+pre-pr: lint mock test
```

Comment:
Added mock to `pre-pr` step. Actually caught a bug where we forgot to update the `CertificateProvider` mock in our previous pr (we moved it from external to internal)

Reply:
Nice, thanks!

Reply:
One thing maybe would be to move the mock before the lint, so that the linter has a chance to consider the regenerate mock code.

Reply:
Good point, reordered mock and lint